### PR TITLE
Presentation data

### PR DESCRIPTION
**Pull request by frederike-ramin**
Create sensible data for use in the final presentation.

### CODE REVIEW

**Review by timbasel — APPROVED** [review]
Looks good to me

**Review by querenker — APPROVED** [review]
I fixed some typos in the rejection mail template. Now it should be fine :)

File: db/sample_data/events.rb [hunk 1]
```diff
@@ -74,7 +74,7 @@ def event_gongakrobatik
       end_date: Date.new(2021, 03, 05)
   )
   Event.new(
-      name: 'Einführung in die Kunst der Gongakrobatik',
+      name: 'Gongakrobatik',
```
Comment (bjrne) [inline db/sample_data/events.rb:77]: Why did you do this? Only for the presentation because long titles don't work well at the moment? If this is the case, please open a bug/technical debt/enhancement issue so we don't forget this (Since titles of that length shouldn't be a problem in the future 🙂 )

File: app/assets/stylesheets/events.css [hunk 2]
```diff
@@ -106,3 +106,7 @@
     height: 150px;
     margin-bottom: 15px;
 }
+
+.text-success{
+    color: #000000;
+}
```
Comment (cmfcmf) [inline app/assets/stylesheets/events.css]: This doesn't look right. Success text should most likely be green.

File: db/sample_data/email_templates.rb [hunk 3]
```diff
@@ -0,0 +1,60 @@
+def email_template_acceptance
+  EmailTemplate.new(
+    status: :acceptance,
+    hide_recipients: false,
+    subject: "Zusage für TODO",
```
Comment (cmfcmf) [inline db/sample_data/email_templates.rb:5]: TODO?

Comment (tobiasduerschmid) [reply db/sample_data/email_templates.rb:5]: TODO schould be changed by the organizer (enter camp name, etc.) This is intended :)

File: db/sample_data/events.rb (hunk 1 above)
Comment (tobiasduerschmid) [reply db/sample_data/events.rb:77]: Just because we do not want to have so huge names in order to keep the UI simple.

File: db/sample_data/email_templates.rb (hunk 3 above)
Comment (cmfcmf) [reply db/sample_data/email_templates.rb:5]: Oh, right. Got it!

**Review by tobiasduerschmid — APPROVED** [review]
Now the texts should be great :)

File: db/sample_data/events.rb (hunk 1 above)
Comment (bjrne) [reply db/sample_data/events.rb:77]: So it is more or less just a presentation thing? Future users might still choose a long name...

File: app/assets/stylesheets/events.css [hunk 4]
```diff
@@ -118,3 +118,7 @@
     height: 150px;
     margin-bottom: 15px;
 }
+
+.text-success{
```
Comment (Baschdl) [inline app/assets/stylesheets/events.css]: Why did you change a css file?

**Review by bjrne — DISMISSED** [review]
Formal block based on @Baschdl's question

File: app/assets/stylesheets/events.css (hunk 4 above)
Comment (tobiasduerschmid) [reply app/assets/stylesheets/events.css]: Because the text color was white and nobody could read it.

File: db/sample_data/events.rb (hunk 1 above)
Comment (tobiasduerschmid) [reply db/sample_data/events.rb:77]: Yes

**Review by bjrne — APPROVED** [review]
🍭
